test/unit/lib/monocle_test.js

```diff
@@ -246,6 +297,44 @@ describe('Monocle API Client', function() {
             }.bind(this));
         });
 
+        it('makes new http request if query changes', function() {
```

Comment:
I'd be great to have a unit test that takes into account when one call has query parameter and one call doesn't have a query parameter